pixeldata/src/lib.rs

```diff
@@ -532,17 +531,16 @@ impl DecodedPixelData<'_> {
 
     /// Retrieve object's rescale parameters.
     #[inline]
-    pub fn rescale(&self) -> Rescale {
-        Rescale {
-            intercept: self.rescale_intercept,
-            slope: self.rescale_slope,
-        }
+    pub fn rescale(&self) -> Vec<Rescale> {
+        zip(&self.rescale_intercept, &self.rescale_slope)
+            .map(|(intercept, slope)| Rescale { intercept: *intercept, slope: *slope })
+            .collect()
     }
 
     /// Retrieve the VOI LUT function defined by the object, if any.
     #[inline]
-    pub fn voi_lut_function(&self) -> Option<VoiLutFunction> {
-        self.voi_lut_function
+    pub fn voi_lut_function(&self) -> Option<&Vec<VoiLutFunction>> {
```

Comment:
Returning `&[T]` instead of `&Vec<T>` would be more idiomatic and enable us to change the inner container type later.

```suggestion
    pub fn voi_lut_function(&self) -> Option<&[VoiLutFunction]> {
```